ambry-store/src/main/java/com.github.ambry.store/BlobStore.java

```diff
@@ -220,6 +220,9 @@ public void start() throws StoreException {
                 taskScheduler, diskIOScheduler, metrics);
         checkCapacityAndUpdateReplicaStatusDelegate();
         logger.trace("The store {} is successfully started", storeId);
+        if (replicaId != null) {
```

Comment:
minor: do this after `started = true` (to avoid race conditions)

Reply:
Also, should `BlobStore` do this or `DiskManager` (since `ReplicaId` can be `null` here)